### ISSUE TITLE
Different results with xgboost and onnxruntime when n_estimators > 1

### ISSUE THREAD

**Issue by shirazzaman** [microsoft/onnxruntime#1585]
**Describe the bug**
Converted a simple (7 float32 predictors) xgboost model to onnx format using onnxmltools.
Loaded the onnx file using onnxruntime and called inference (sess.run) on a test dataframe.

The result from onnxruntime matches original prediction when n_estimators = 1 but widely differs when n_estimators > 1

**Urgency**
If there are particular important use cases blocked by this or strict project-related timelines, please share more information and dates. If there are no hard deadlines, please specify none.
none

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Mac OS 10.14.6
- ONNX Runtime installed from (source or binary):  python binary (pip install)
- ONNX Runtime version: 0.5.0
- Python version:   3.6.7
- Visual Studio version (if applicable):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**To Reproduce**
Describe steps/code to reproduce the behavior:
1. Generate a random dataframe (i used 1000 records with 7 float32 predictors)
2. train xgboost.XBRegressor with n_estimators = 2
3. use onnxmltools.convert_xgboost to convert to onnx format
4. save the model using onnxmtools.save_model
5. sess = rt.InferenceSession('onnxg_demo_new.onnx', None)
6. input_name = sess.get_inputs()[0].name
7. label_name = sess.get_outputs()[0].name
8. pred_onx = sess.run([label_name], {input_name: test.to_numpy()})[0]

**Expected behavior**
I printed the prediction from xgboost before converting to onnx format and the score was 0.4967, and i uses sess.run to predict the score with the same test set with onnxruntime and i got 0.5028. I expect the results to be the same if they are same model.

**Screenshots**

**Additional context**
Add any other context about the problem here. If the issue is about a particular model, please share the model details as well to facilitate debugging.

**Comment by prabhat00155** [comment]
https://github.com/onnx/onnx/issues/2228

**Comment by prabhat00155** [comment]
https://github.com/onnx/onnxmltools/pull/336

**Comment by prabhat00155** [comment]
Closing this as issue explained in onnx/onnx#2228 has been fixed. If you still see the issue related to n_estimators, feel free to reopen the issue with a sample of the script you used.
